app/controllers/comments_controller.rb

```diff
@@ -1,4 +1,5 @@
 class CommentsController < ApplicationController
+before_action :find_commentable
```

Comment:
Do you need this on each action here?

Reply:
I guess only on Create, is there something like only: :create that specifies?

Reply:
Yes, there are options for specifying which actions these can be used on. Docs here: https://guides.rubyonrails.org/action_controller_overview.html#filters